doc/articles/features/VisibleBoundsPadding.md

```diff
@@ -42,6 +42,16 @@ Usage is as follows:
 </UserControl>
 ```
 
-**Note: 
+#### Note: 
 - This behavior applies the greater of the existing padding or the calculated padding set on the element it is attached to.
-**
+
+
+## See it in action
+
+![image](https://user-images.githubusercontent.com/36631443/68246487-defa0900-ffe6-11e9-8e5b-2beaea7bdc51.png)
```

Comment:
It would be best if the images were local to the repository, in the assets folder.